bin/prettier.js

```diff
@@ -171,17 +171,15 @@ if (stdin) {
 } else {
   eachFilename(filepatterns, filename => {
     fs.readFile(filename, "utf8", (err, input) => {
-      if (write) {
-        console.log(filename);
-      }
-
       if (err) {
```

Comment:
Errors in 
file read https://github.com/jlongster/prettier/blob/master/bin/prettier.js#L179 
format https://github.com/jlongster/prettier/blob/master/bin/prettier.js#L189 and 
write https://github.com/jlongster/prettier/blob/master/bin/prettier.js#L199 already contains filename, do we really need to add extra line with filename ?